website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -39,6 +39,9 @@ Documents in this section go over steps to run different types of React Native r
 
 - Add and commit the extra file that got created at `sdks/hermes/.hermesversion`.
 
+- Update packages in the monorepo by running `yarn bump-all-updated-packages`. All the package bumps should be a patch. We need to publish the latest package changes, so they will be included in a new RC. Read more about the script and how they work [here](./release-updating-packages).
+- Push the commit created by the previous command to the `0.x-stable` branch.
```

Comment:
```suggestion
- Push the commit created by the previous command to the `0.XX-stable` branch so that CI will take care of releasing the new versions on npm.
```